Sources/MySQL/Database.swift

```diff
@@ -74,6 +78,7 @@ public final class Database {
     private let socket: String?
     private let flag: UInt
     private let encoding: String
+    public static var fractionalSecondsDigits: Int = 0
```

Comment:
I have updated the PR to have the new per-database-instance control over the subSecond resolution on SQL TIME types.